spk/python311/Makefile

```diff
@@ -103,10 +98,20 @@ endif
 
 include ../../mk/spksrc.common.mk
 
-# # Enable debug_info builds for DSM6
-# ifeq ($(call version_lt, ${TCVERSION}, 7),1)
-# GCC_DEBUG_INFO := 1
-# endif
+# Enable debug_info symgols for all archs
+GCC_DEBUG_INFO := 1
+
```

Comment:
I suppose you will deactivate `GCC_DEBUG_INFO` before merging this PR?

Reply:
Absolutely, I just wanted it to run once first.  Also, python311 update is overdue anyway.

Reply:
> python311 update is overdue anyway.

yes, need Pillow 10.1.0 and others for homeassistant update....

Reply:
yikes... this will be an interesting one.  I will definitively need some help, at least for testing.  Will be looking into it upon my return from vacation.

Reply:
The challenge for me will be the `ha_av` wheel, i.e. build home assistant with both build dependencies of python311 and ffmpeg,
So this will need some time, no need to hurry, ....